lib/manager/gradle-wrapper/artifacts.ts

```diff
@@ -3,25 +3,35 @@ import Git from 'simple-git/promise';
 import { resolve } from 'path';
 import * as fs from 'fs-extra';
 import { logger } from '../../logger';
-import { UpdateArtifact, UpdateArtifactsResult } from '../common';
+import {
+  UpdateArtifact,
+  UpdateArtifactsConfig,
+  UpdateArtifactsResult,
+} from '../common';
 import { exec, ExecOptions } from '../../util/exec';
-import { readLocalFile } from '../../util/fs';
 import { platform } from '../../platform';
 import { VERSION_REGEX } from './search';
-import { gradleWrapperFileName, prepareGradleCommand } from '../gradle/index';
+import { gradleWrapperFileName, prepareGradleCommand } from '../gradle';
+import { readLocalFile } from '../../util/fs';
 
 async function addIfUpdated(
+  config: UpdateArtifactsConfig,
   status: Git.StatusResult,
-  projectDir: string,
   fileProjectPath: string
 ): Promise<UpdateArtifactsResult | null> {
   if (status.modified.includes(fileProjectPath)) {
-    const filePath = resolve(projectDir, `./${fileProjectPath}`);
+    const rawFileContents = await readLocalFile(fileProjectPath);
+    let fileContents;
+    if (fileProjectPath.endsWith('.jar')) {
+      fileContents = rawFileContents;
+    } else {
+      fileContents = rawFileContents.toString('utf8');
+    }
     return {
       artifactError: null,
       file: {
         name: fileProjectPath,
-        contents: await readLocalFile(filePath),
+        contents: fileContents,
```

Comment:
```suggestion
        contents: await readLocalFile(fileProjectPath),
```

Reply:
@viceice Your suggestion is incorrect, because I need to read in a byte array only for the `gradle-wrapper.jar` file. All other files are text files, so they must be read using the `utf8` encoding.